packages/react-components/react-nav-preview/src/components/Nav/useNav.ts

```diff
@@ -11,18 +20,67 @@ import type { NavProps, NavState } from './Nav.types';
  * @param props - props from this instance of Nav
  * @param ref - reference to root HTMLElement of Nav
  */
-export const useNav_unstable = (props: NavProps, ref: React.Ref<HTMLElement>): NavState => {
+export const useNav = (props: NavProps, ref: React.Ref<HTMLElement>): NavState => {
```

Comment:
We should be more specific with our ref types
```suggestion
export const useNav = (props: NavProps, ref: React.Ref<HTMLDivElement>): NavState => {
```